api/v1/openlibertyapplication_types.go

```diff
@@ -147,6 +147,9 @@ type OpenLibertyApplicationSpec struct {
 
 	// +operator-sdk:csv:customresourcedefinitions:order=32,type=spec,displayName="Topology Spread Constraints"
 	TopologySpreadConstraints *OpenLibertyApplicationTopologySpreadConstraints `json:"topologySpreadConstraints,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:order=33,type=spec,displayName="Generation"
+	Generation *int32 `json:"generation,omitempty"`
```

Comment:
Let's use the generation field under metadata of the CR, which is automatically incremented when the CR's spec changes. That'll avoid user having to change the field manually.

Reply:
Removed this extra field